app/views/http_proxies/_form.html.erb

```diff
@@ -1,3 +1,7 @@
+<%= alert :header => '', :class => 'alert-warning', :close => false,
+          :text => _("Please note that these HTTP proxy configurations are only used for requests
+                      from foreman or foreman plugins. System-wide proxies must be configured at operating system level.") %>
+
```

Comment:
Isn't this still an alert box? I think just a text that matches the other text on the welcome page is what we want.

Reply:
The recommendation from @xprazak2 was, that the warning (the alert box) should be only on the "new/edit" page and not on the welcome page.

My suggestions is, to use the alert box in the new/edit page. Otherwise a lot of users will not recognize the warning.